### PR TITLE
add sorting to semantic workshop for service bodies and formats

### PR DESCRIPTION
**Pull request by pjaudiomv**

### CODE REVIEW

File: main_server/semantic/bmlt_semantic.js [hunk 1]
```diff
@@ -981,6 +986,18 @@ BMLTSemantic.prototype.createServiceBodyList = function (
...
 };
 
+/*******************************************************************************************/
+/**
+ \brief Sort By Key
+ */
+/*******************************************************************************************/
+BMLTSemantic.prototype.sortByKey = function sortByKey(array, key) {
+    return array.sort(function(a, b) {
+        var x = a[key].toLowerCase(); var y = b[key].toLowerCase();
```
Comment (jbraswell) [inline main_server/semantic/bmlt_semantic.js]: Can you just use strcasecmp here?

File: main_server/semantic/bmlt_semantic.js [hunk 2]
```diff
@@ -981,6 +986,18 @@ BMLTSemantic.prototype.createServiceBodyList = function (
     };
 };
 
+/*******************************************************************************************/
+/**
+ \brief Sort By Key
+ */
+/*******************************************************************************************/
+BMLTSemantic.prototype.sortByKey = function sortByKey(array, key) {
```
Comment (jbraswell) [inline main_server/semantic/bmlt_semantic.js]: You don’t have to name the function twice here. You can just assign sortByKey to function(...) { ... }

Comment (pjaudiomv) [reply main_server/semantic/bmlt_semantic.js]: oh yes thanks, a little redundant eh

File: main_server/semantic/bmlt_semantic.js (hunk 1 above)
Comment (pjaudiomv) [reply main_server/semantic/bmlt_semantic.js]: I was unaware javascript had such a function, ill check it out

**Comment by pjaudiomv** [comment]
oh deer, the semantic is not happy looks like from the changes made the other day
https://latest.aws.bmlt.app/main_server/semantic/
check out console

**Pull request merged**